src/inmanta/server/agentmanager.py

```diff
@@ -910,7 +911,7 @@ async def _ensure_agents(self, env: data.Environment, agents: List[str], restart
             raise ShutdownInProgress()
 
         agent_map: Dict[str, str]
-        agent_map = await env.get(data.AUTOSTART_AGENT_MAP)
+        agent_map = cast(Dict[str, str], await env.get(data.AUTOSTART_AGENT_MAP))  # we know the type of this map
```

Comment:
Same comment here (and below) with regards to the cast.

Reply:
you mean that I should add the comment in those places as well?